examples/factor_model.py

```diff
@@ -1,48 +1,51 @@
-from cvxpy import *
-from ncvx import *
+import cvxpy as cp
+from cvxpy import norm, Variable, diag, sum_squares, Minimize, trace, Problem, Parameter, pos
```

Comment:
Use ``cp.Variable`` etc instead of separately importing.

Reply:
I was shortening a few examples down with the separate ``from``, but I'll remove those in any examples where I used it.